examples/seismic/abc_methods/03_pml.ipynb

```diff
@@ -0,0 +1,1423 @@
+{
```

Comment:
<p> is the one proposed by Grote and Sim + link ?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I don't understand your question here. Can you add more details?

Reply:
I mean to add the reference to the method.

Reply:
I don't find other site that contains this paper, just in arxiv, because this I put this link.